engines/account/app/controllers/account/api/v1/login_controller.rb

```diff
@@ -16,7 +17,7 @@ def login
         private
 
         def login_params
-          params.permit(:email, :password)
+          params.permit(:username, :email, :password)
```

Comment:
Maybe instead of passing both `:username` and `:email` you could just pass `:login` parameter, that can be either an email or an username. What do you think?